CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
 cmake_minimum_required(VERSION 2.8)
+cmake_policy(SET CMP0042 NEW)
```

Comment:
Is it better to bump the minimum required version to a more recent one?  2.8 is really old now.